index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Canvas</title>
+</head>
+<body>
+  <div class="body-wrapper">
```

Comment:
actually you could apply styles of class body-wrapper directly for the "body" element, this wrapper seems useless